shopfloor/services/single_pack_transfer.py

```diff
@@ -207,7 +207,8 @@ def _set_destination_and_done(self, move, scanned_location):
         # when writing the destination on the package level, it writes
         # on the move lines
         move.move_line_ids.package_level_id.location_dest_id = scanned_location
-        move._action_done()
+        # FIXME: '_sf_no_backorder' was not passed in the context, is it expected?
```

Comment:
Perhaps. Maybe. I don't know.
Originally, this scenario was meant to move a picking with a single package, the behavior for other uses cases are undefined (to my knowledge).